docs/articles/roadmap/README.md

```diff
@@ -15,28 +16,31 @@ Please vote on the features you want us to implement first to help us prioritize
 
 Feature                                                                                           | Status
 ------------------------------------------------------------------------------------------------- | -------------------
-[Testing in Multiple Browser Windows](https://github.com/DevExpress/testcafe/issues/912)          | **In Progress**
-[API Testing](https://github.com/DevExpress/testcafe/issues/4288)                                 | Planned
+[Testing in Multiple Browser Windows](https://github.com/DevExpress/testcafe/issues/912)          | **Implemented**
+[Selector Debug Panel](https://github.com/DevExpress/testcafe/issues/3244)                        | **In Progress**
+[HTTP API Testing](https://github.com/DevExpress/testcafe/issues/4288)                            | Planned
+[Cookie Management](https://github.com/DevExpress/testcafe/issues/5964)                           | Planned
 
 ## API
 
 Feature                                                                                           | Status
 ------------------------------------------------------------------------------------------------- | -------------------
-[Configuration File](https://github.com/DevExpress/testcafe/issues/3131)                          | **Implemented**
-[Screenshots of the Entire Page](https://github.com/DevExpress/testcafe/issues/1520)              | **Implemented**
-[Browser Information in Test Code](https://github.com/DevExpress/testcafe/issues/481)             | **Implemented**
+[Dedicated *Scroll* Action](https://github.com/DevExpress/testcafe/issues/5965)                   | Planned
 
 ## Debugging
 
 Feature                                                                                           | Status
 ------------------------------------------------------------------------------------------------- | -------------------
-[Video Recording](https://github.com/DevExpress/testcafe/issues/2151)                             | **Implemented**
-[Live Mode](https://github.com/DevExpress/testcafe/issues/3215)                                   | **Implemented**
-[Selector Debug Panel](https://github.com/DevExpress/testcafe/issues/3244)                        | **In Progress**
+[Interactive Demo Center](https://github.com/DevExpress/testcafe/issues/5966)                     | Planned
```

Comment:
```suggestion
[Interactive Demo Page](https://github.com/DevExpress/testcafe/issues/5966)                     | Planned
```

Reply:
We already have an [interactive demo page](https://devexpress.github.io/testcafe/example/), don't we?

Reply:
It's an example page, not a demo page. It doesn't demonstrate anything on its own.